.github/workflows/validate-python-subdirectory.yaml

```diff
@@ -0,0 +1,51 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python.
+# Use this workflow if your Python code is in a subdirectory of a non-Python repository.
```

Comment:
⛏️ this was presumably copied from our python template. I think the comment can be updated or removed as no one will actually use this workflow elsewhere now that it's in DAP.